File: BOJ(Java 공부용)/10870.java
import java.io.*;
public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        String arr[] = br.readLine().split(" ");
        int n= Integer.parseInt(arr[0]);
        bw.write(String.valueOf(go(n)));
        bw.flush();
    }

    private static int go(int n) {
        if(n==0) return 0;
        if(n==1) return 1;
        return go(n-1)+go(n-2);
    }
}

File: BOJ(Kotlin)/10870.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.util.*
import kotlin.math.max
import kotlin.math.min

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val bw = BufferedWriter(OutputStreamWriter(System.out))

    val st = StringTokenizer(br.readLine())
    val n = st.nextToken().toInt()
    val res = fibo(n)
    bw.write(res.toString()+"\n")
    bw.flush()
    bw.close()
}
fun fibo(n : Int) : Int {
    if (n == 0 || n == 1) return n
    else return fibo(n-1 )+ fibo(n-2)
}